circle_suck/tests/test_utils.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from base.utils import *
+
+class TestCycleFinder(TestCase):
+	def test_single_cycle(self):
+		self.assertEqual(find_all_connected({1:[2],2:[1,5],3:[4],4:[3,5],5:[6],6:[7],7:[8],8:[6,9],9:[]}), [[8, 6, 7]])
```

Comment:
make these a little bit pretty pls, with comments explaining what you're testing in each case.
```
season = {
    1: [2],
    2: [1, 5],
    ...
}
self.assertEqual(find_all_connected(season), [[8,6,7]])
```